src/RiveQtQuickItem/rhi/texturetargetnode.cpp

```diff
@@ -171,7 +171,7 @@ void TextureTargetNode::updateViewport(const QRectF &bounds, QRhiTexture *displa
     m_blendVerticesDirty = true;
 }
 
-void TextureTargetNode::prepareRender(QRhiCommandBuffer *cb)

```

Comment:
can you remove the texturetargetnode changes from this PR? I dont think they belong here and they mess with my upcoming pipeline changes